src/dolphin/playkey.ts

```diff
@@ -8,7 +8,17 @@ import type { DolphinInstallation } from "./install/installation";
 
 export async function writePlayKeyFile(installation: DolphinInstallation, playKey: PlayKey): Promise<void> {
   const keyPath = await findPlayKey(installation);
-  const contents = JSON.stringify(playKey, null, 2);
+  const contents = JSON.stringify(
```

Comment:
this is a bit unnecessary, just change the type signature to

`playKey: Omit<PlayKey, "rulesAccepted">`

Reply:
I didn't want to do this because if we add more fields, we'll have to remember to add them to this list.